### ISSUE TITLE
Is disabling the GPU sandbox still necessary?

### ISSUE THREAD

**Issue by DerProfessor** [flathub/com.discordapp.Discord#304]
Hi there,
I was wondering if disabling the GPU sandbox is still necessary for NVIDIA GPUs while running a Wayland compositor. The description of PR #190 talks about a possible security implication when disabling the GPU sandbox.
I tested the rendering problems by skipping the part of the script that disables electron's GPU sandbox ([this line](https://github.com/flathub/com.discordapp.Discord/blob/5bb8e340ee542fcbad0c1192e32df912bf94f979/discord.sh#L20)). It seems that the internal upgrades of Discord fixed this rendering issue (maybe by upgrading the electron version?). I was running the client with the following specs:

Fedora 37, KDE Plasma Wayland 5.27.4.1
Nvidia driver 530.30.02
Flatpak Discord wrapper commit [5bb8e34](https://github.com/flathub/com.discordapp.Discord/commit/5bb8e340ee542fcbad0c1192e32df912bf94f979)

**Comment by TingPing** [comment]
I'd be interested in somebody testing it on GNOME as well.

**Comment by lionirdeadman** [comment]
There is already a PR to re-enable the GPU sandbox (https://github.com/flathub/com.discordapp.Discord/pull/295). Unfortunately, I have not been successful at finding people to test it.

**Comment by guihkx** [comment]
The GPU sandbox was re-enabled for NVIDIA on Wayland a year ago, by PR #332.

Later, however, that NVIDIA-specific check was also removed by https://github.com/flathub/com.discordapp.Discord/commit/5fd294d592a023b00213ce8745235a8d73207c40.

TL;DR: We don't disable the GPU sandbox anymore.
